Makefile.in

```diff
@@ -75,7 +75,7 @@ mana: mana_prereqs
 	$(MAKE) mana_part2
 mana_part2: default \
 	bin/mana_launch bin/mana_restart bin/mana_coordinator bin/mana_status
-	cd contrib/mpi-proxy-split && make install && make -j tests
+	cd contrib/mpi-proxy-split && make mana && make -j tests
```

Comment:
I think we don't need to add `mana` in the Makefiles under `mpi-proxy-split` since this directory implies we're building for mana.

Reply:
sorry, could you clarify what you mean by this?

Reply:
I think we can just use `make` to make it simpler, `mana` is the default if you want to keep it.